backend/tests/unit/agents/test_iteration_controllers.py

```
@@ -18,6 +18,7 @@
 )
 from tarsy.models.constants import IterationStrategy
 from tarsy.models.processing_context import AvailableTools, ChainContext, StageContext
+from tarsy.models.unified_interactions import LLMConversation, LLMMessage, MessageRole
```

Ruff (0.12.2)

`tarsy.models.unified_interactions.LLMMessage` imported but unused

Remove unused import: `tarsy.models.unified_interactions.LLMMessage`

(F401)